rules/pulpos/default_rules.mk

```diff
@@ -317,16 +319,22 @@ $(TARGET_BUILD_DIR)/stdout:
 $(TARGET_BUILD_DIR)/fs:
 	mkdir -p $@
 
-
+ifeq '$(pulp_chip)' 'pulp_cluster'
+run:
+	vsim $(vsim-flags) -do "set  VSIM_PATH $(VSIM_PATH); source $(VSIM_PATH)/scripts/start.tcl"
+else
```

Comment:
Is this completely separate target for pulp_cluster required? Is it possible to use the same flow as below? If not, I suggest to keep as much as possible aligned (e.g. using `$(VSIM)` instead of `vsim` to start vsim).

Reply:
This one is a bit tricky: I didn't check the existing flow since the pulp cluster tb is very much different from pulp's tb. I could dig into the "baseline" scripts and modify the cluster's testbench to become compatible. However, I honestly think that the `pulp` flow is a bit convoluted and overkill for our needs, aligning new code to a legacy version maybe it's not the best approach, but I see why avoiding splitting into different targets.

Reply:
These is the `default_rules.mk` file which should apply, well, to things that should be default. Custom stuff should go into `pulp_cluster.mk`. You can just overwrite the run: target in pulp_cluster.mk with whatever else you want it to be.

Reply:
Yeah, you are right. This makes more sense!

Reply:
As you suggested, I am now overwriting the `run` target within the `pulp_cluster.mk`